src/services/db/RepoService.ts

```diff
@@ -804,8 +804,6 @@ export default class RepoService extends GitHubService {
     sessionData: any,
     shouldMakePrivate: any
   ): Promise<any> {
-    return await super.changeRepoPrivacy(sessionData, {
-      shouldMakePrivate,
```

Comment:
@alexanderleegs the API here seemed inconsistent, but I think I might be at the wrong here. happy to huddle this to shoulder check, but looking at the call sites should be able to just pass in boolean